js/src/ui/SelectionList/selectionList.css

```diff
@@ -58,7 +79,3 @@
   filter: grayscale(10%);
   opacity: 0.75;
```

Comment:
? Not sure it is for this PR? 

1. As-is in master currently?
2. Remember the original complaint, it looked "disabled" so users didn't know what to do.

Agreed though, I'm "ok" but not "happy" with the selections, warrants a bit more playing with the colors, borders & opacity.